src/main/java/WebRTCTestRunner.java

```diff
@@ -0,0 +1,84 @@
+
+import java.net.URL;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+public class WebRTCTestRunner implements Runnable {
+
+	
+	public static final String USERNAME = System.getenv("BROWSERSTACK_USERNAME");
+   public static final String ACCESS_KEY = System.getenv("BROWSERSTACK_ACCESS_KEY");
+
+    public static final String HUB_URL = "https://" + USERNAME + ":" + ACCESS_KEY + "@hub-cloud.browserstack.com/wd/hub";
+    WebDriver driver = null;
+    String roomId = null;
+    boolean joinExisting = false;
+    int userSelection = 0;
+    long duration = 15000;
+    ChromeOptions options = null;
+    public WebRTCTestRunner(ChromeOptions options, String roomId, boolean joinExisting, long duration, int userSelection) {
+        this.options = options;
+        this.roomId = roomId;
+        this.joinExisting = joinExisting;
+        this.duration = duration;
+        this.userSelection = userSelection;
+    }
+
+    public void run() {
+
+        try {
+        	
+            //Creating Remote WebDriver based on the capabilites defined in WebRTCConf.java.
+            driver = new RemoteWebDriver(new URL(HUB_URL), options);
+            
+            //Creating new WebRTC Room with generated roomID
+            if(!joinExisting) {
+            	
+            	if(userSelection != 3) {
+                	driver.manage().window().maximize();
+                }
+            	
+                driver.get("https://appr.tc/");
```

Comment:
have string variables like these defined at the top of the class as distinct vars

Reply:
done